mteb/evaluation/MTEB.py

```diff
@@ -297,7 +297,19 @@ def run(
 
         # Create output folder
         if output_folder is not None:
-            pathlib.Path(output_folder).mkdir(parents=True, exist_ok=True)
+            # check for model revision. For SentenceTransformers, this is available from sentence-transformers==3.0.0.
+            # if not found, check for supplied model revision.
+            if hasattr(model, "revision"):
+                model_revision = model.revision
+            else:
+                try:
+                    model_revision = model.model_card_data.base_model_revision
+                except AttributeError:
+                    logger.warning("Please supply model revision.")
+                    return []
```

Comment:
I would just default to "model_name/no_revision_available/results"

Reply:
Added